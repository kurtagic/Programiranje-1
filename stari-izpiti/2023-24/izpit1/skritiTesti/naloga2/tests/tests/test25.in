1 2 39 457
0 17 351
0 17 419
0 12 167
1 27 301
0 34 385
0 12 469
0 34 12
0 12 254
1 12 153
0 27 378
0 37 322
1 22 322
0 35 320
1 8 301
0 12 438
0 1 150
1 27 281
0 1 519
0 1 353
0 25 315
0 34 354
0 38 106
0 22 250
1 35 184
0 13 393
0 22 244
1 5 21
0 29 519
1 35 47
0 13 461
1 0 204
1 12 504
0 22 174
0 0 284
0 30 128
0 34 173
0 25 491
1 1 433
1 36 515
0 5 478
1 34 497
1 1 194
0 1 501
0 36 219
1 38 386
0 35 467
0 1 223
1 20 355
0 25 188
1 5 490
0 3 304
1 35 478
1 27 184
0 23 151
0 37 497
1 38 493
0 3 41
0 12 298
0 11 156
1 17 37
1 6 344
0 9 8
0 4 357
1 5 180
1 8 181
0 7 5
1 8 248
1 38 50
0 34 48
1 20 514
0 20 343
1 4 76
0 30 364
1 24 524
0 16 253
0 32 480
0 13 325
1 36 248
1 30 87
1 20 292
1 12 401
1 34 166
0 3 400
1 37 392
0 10 50
1 1 137
0 10 268
1 10 130
0 18 71
0 36 167
0 16 250
0 38 86
1 20 523
1 3 26
0 38 359
0 13 389
1 12 315
1 34 68
0 26 152
1 30 47
1 33 498
0 37 123
1 36 436
0 1 283
0 10 486
1 0 391
0 14 214
0 30 134
0 22 143
0 5 268
1 20 337
1 5 510
0 16 206
1 36 10
0 30 523
0 35 485
1 19 162
1 30 162
1 26 300
0 12 138
0 25 254
1 5 343
1 32 347
1 37 6
0 37 272
1 36 187
1 34 77
1 24 159
0 22 274
0 23 332
1 20 7
1 12 342
0 12 297
0 16 403
0 6 207
0 26 37
1 3 352
0 9 189
1 0 25
1 5 263
1 4 33
0 12 409
0 6 491
0 26 128
1 9 256
0 32 358
0 1 39
1 21 153
0 22 328
1 34 309
1 18 422
1 37 82
0 11 256
0 0 267
0 17 228
0 11 191
0 0 68
1 32 175
0 24 218
0 23 515
1 30 518
1 19 229
1 18 74
1 32 106
1 1 503
1 19 257
0 1 3
1 27 379
0 17 369
0 1 103
0 16 321
1 38 297
0 10 476
0 32 108
0 19 193
0 3 439
0 16 226
1 9 193
1 19 488
0 16 274
1 20 131
0 10 430
0 31 310
1 13 348
1 0 69
1 5 233
1 26 303
0 0 519
0 3 361
1 6 326
0 24 124
0 9 100
0 23 509
0 36 248
1 28 419
0 5 200
1 14 27
1 8 363
0 32 398
1 4 18
0 37 103
1 1 411
0 15 347
0 21 269
1 26 387
1 19 338
0 35 480
0 9 231
1 35 187
1 35 206
0 33 472
1 38 290
0 1 34
1 3 275
0 29 273
1 37 143
0 30 199
0 12 13
0 13 517
1 12 42
0 3 335
0 1 47
0 37 266
1 32 322
0 10 224
1 18 213
0 24 120
1 24 184
0 12 407
0 24 265
0 11 114
0 31 23
1 17 325
1 22 125
1 36 186
0 20 209
0 24 510
0 18 320
1 21 361
1 4 199
1 0 486
0 22 231
0 16 457
1 31 507
0 21 404
1 34 439
0 20 417
0 7 411
1 30 347
1 32 191
1 23 460
0 29 430
1 32 54
1 32 61
0 27 228
0 5 136
1 31 442
0 12 423
1 20 319
1 35 175
1 24 91
0 16 185
0 4 126
0 19 356
0 12 355
0 8 348
1 36 276
0 33 437
0 0 231
1 31 489
1 31 368
1 1 503
1 24 81
1 9 455
0 8 104
0 16 237
1 26 378
1 21 382
0 36 347
1 3 406
1 17 104
0 33 260
0 9 119
0 14 470
1 12 471
1 0 70
1 36 326
1 31 70
0 4 118
1 26 444
1 3 483
1 24 409
1 15 474
1 5 264
1 34 177
0 1 104
0 37 331
0 33 47
1 12 348
0 14 208
1 26 102
1 31 9
0 21 161
1 25 443
1 31 425
1 24 373
1 5 89
0 19 164
0 8 42
1 34 25
0 25 223
0 31 1
1 22 363
1 26 389
1 9 530
1 1 206
1 16 223
1 9 253
1 19 60
0 9 467
0 8 386
0 10 163
1 30 56
1 13 451
1 29 401
0 23 352
0 14 502
0 25 48
0 30 189
0 33 477
0 2 41
1 24 422
0 33 22
1 38 346
1 32 392
0 26 183
0 32 137
0 0 384
1 1 147
0 27 249
0 15 326
0 2 62
1 30 424
0 12 71
1 12 241
0 1 409
1 1 19
0 10 449
0 38 141
0 13 58
0 34 468
0 27 432
1 14 214
1 22 425
1 38 284
1 31 136
1 36 159
0 35 145
1 6 11
1 16 356
0 23 527
1 12 363
0 18 273
0 38 85
0 0 73
1 12 396
1 32 151
0 8 182
1 31 319
0 20 252
0 38 101
0 33 206
0 33 96
0 6 355
1 35 284
1 8 455
1 21 343
0 13 287
0 11 14
0 36 98
0 36 253
1 18 509
0 1 338
0 23 355
0 36 465
1 27 22
1 6 380
0 17 477
0 2 480
0 26 261
0 35 397
1 23 232
1 4 101
1 37 67
0 2 256
0 9 380
1 8 469
0 37 364
1 19 243
0 24 90
0 10 523
1 20 182
1 1 313
0 30 96
1 24 342
1 13 511
1 33 7
1 30 197
1 30 439
1 26 267
1 38 275
0 18 160
0 5 380
1 16 419
0 29 276
1 1 390
0 10 71
1 35 484
1 24 343
1 16 271
0 35 421
0 32 517
0 37 60
1 32 166
0 13 477
1 33 233
1 32 151
0 23 307
0 0 361
1 6 435
0 1 199
1 5 194
0 26 354
1 30 133
1 18 434
0 1 141
0 32 364
1 3 195
0 1 152
1 35 286
1 3 488
0 4 195
1 35 356
0 11 185
0 30 377
0 17 365
0 38 391
1 1 59
1 37 112
1 35 108
1 5 131
1 24 140
0 37 501
1 3 15
1 18 163
0 5 240
0 4 42
