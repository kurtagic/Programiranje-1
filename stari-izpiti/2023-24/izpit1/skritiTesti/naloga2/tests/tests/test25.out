[519, 519, 480, 488, 357, 510, 491, 411, 469, 530, 523, 256, 504, 517, 502, 474, 457, 477, 509, 488, 523, 404, 425, 527, 524, 491, 444, 432, 419, 519, 523, 507, 517, 498, 497, 485, 515, 501, 493]
